core/templates/dev/head/pages/exploration_editor/editor_tab/customize_interaction_modal_directive.html

```diff
@@ -29,12 +29,13 @@ <h3 ng-if="stateInteractionIdService.displayed && (customizationModalReopened ||
 
   <form ng-if="stateInteractionIdService.displayed && hasCustomizationArgs" name="form.schemaForm" class="protractor-test-interaction-editor">
     <div ng-repeat="customizationArgSpec in customizationArgSpecs track by $index">
-      <div class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
+      <div ng-if="customizationArgSpec.schema.type !== 'bool'" class="oppia-interaction-customization-label"><[customizationArgSpec.description]></div>
       <div>
```

Comment:
Do you need this containing div?

Reply:
That div only have the description.That div closes in the same line.

Reply:
I'm talking about line 33...

Reply:
It's already there,but I think removing it does not affect anything.